lib/Backend/LowerMDShared.cpp

```diff
@@ -8745,17 +8758,18 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
                 Assert(src->IsFloat32());
                 zero = IR::MemRefOpnd::New((float*)&Js::JavascriptNumber::k_Float32Zero, TyFloat32, this->m_func, IR::AddrOpndKindDynamicFloatRef);
             }
+
+            IR::LabelInstr * skipRoundSd = IR::LabelInstr::New(Js::OpCode::Label, this->m_func);
             if(instr->m_opcode == Js::OpCode::InlineMathRound)
             {
+                IR::LabelInstr* addHalfToRoundSrcLabel = IR::LabelInstr::New(Js::OpCode::Label, this->m_func);
+                IR::LabelInstr* skipAddHalfLabel = IR::LabelInstr::New(Js::OpCode::Label, this->m_func);
+
                 if(instr->ShouldCheckForNegativeZero())
```

Comment:
I am not a 100% sure, but I think it has to do with whether the result of round is used later. @LouisLaf , could you take a look at the code inside the `if (instr->ShouldCheckForNegativeZero()' and see whether it might be a problem if the condition evaluates to false?